packages/report-server/src/reportBuilder/functions/utils.ts

```diff
@@ -28,3 +29,18 @@ export const dateStringToPeriod = (period: string, targetType: string): string =
   }
   return baseDateStringToPeriod(period, targetType);
 };
+
+export const formatAsFractionAndPercentage = (numerator: number, denominator: number) => {
```

Comment:
Could we get some unit tests for this plz! 🙏

Reply:
Sure. Unit test has been implemented.